### ISSUE TITLE
Searches using hyphens on HPO browser - results that don't match label exactly?

### ISSUE THREAD

**Issue by peupeubangbang** [TheJacksonLaboratory/hpo-web#290]
Not sure if this is possible but would there be any possibility in relaxing the need for hyphens in searches for matching disease term labels? 

E.g. if I'm looking for septo-optic dysplasia HPOA results...

"septo-optic dysplasia" will only return ORPHA:3157
https://hpo.jax.org/browse/search?q=septo-optic&navFilter=all

"septooptic dysplasia" only returns OMIM:182230
https://hpo.jax.org/browse/search?q=septooptic&navFilter=all

"septo optic dysplasia" returns no disease results
https://hpo.jax.org/browse/search?q=septo%20optic%20dysplasia&navFilter=all

Thinking more broadly, some general leniency in not having to have an exact match for phenotype labels or other sorts double barelled terms with spaces between words etc might be useful :)

Thanks for considering!

**Comment by iimpulse** [comment]
@peupeubangbang Thanks for the report, I will look into this as soon as I can.

**Comment by iimpulse** [comment]
Deploying with 2.0.4 1.08 oan
